examples/some-verifier/src/main.rs

```diff
@@ -0,0 +1,233 @@
+    )]
+    db_config: tokio_postgres::Config,
+    #[clap(
+        long = "log-level",
+        default_value = "info",
+        help = "Maximum log level.",
+        env = "SOME_VERIFIER_LOG_LEVEL"
+    )]
+    log_level: tracing_subscriber::filter::LevelFilter,
+    #[clap(
+        long = "api-port",
+        default_value = "8080",
+        help = "Port of the verification check API.",
+        env = "SOME_VERIFIER_API_PORT"
+    )]
+    api_port: u16,
+}
+
+#[derive(Clone)]
+struct AppState {
+    client: reqwest::Client,
+    tdlib_client: Arc<rust_tdlib::client::Client<TdJson>>,
+    discord_bot_token: Arc<String>,
+    database: Arc<Database>,
+}
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    let app = App::parse();
+
+    {
+        let log_filter =
+            tracing_subscriber::filter::Targets::new().with_target(module_path!(), app.log_level);
+        use tracing_subscriber::prelude::*;
+        tracing_subscriber::registry()
+            .with(tracing_subscriber::fmt::layer())
+            .with(log_filter)
+            .init();
+    }
+    tdjson::set_log_verbosity_level(1);
+
+    tracing::info!("Connecting to database...");
+    let database = Database::connect(app.db_config).await?;
+
+    tracing::info!("Starting Telegram client...");
+    let tdlib_params = TdlibParameters::builder()
+        .database_directory("tddb")
+        .use_test_dc(false)
+        .api_id(app.telegram_api_id)
+        .api_hash(app.telegram_api_hash)
+        .system_language_code("en")
+        .device_model("Desktop")
+        .system_version("Unknown")
+        .application_version(env!("CARGO_PKG_VERSION"))
+        .enable_storage_optimizer(true)
+        .build();
+
+    let tdlib_client = rust_tdlib::client::Client::builder()
+        .with_tdlib_parameters(tdlib_params)
+        .with_client_auth_state_handler(ConsoleClientStateHandlerIdentified::new(
+            ClientIdentifier::BotToken(app.telegram_bot_token),
+        ))
+        .build()?;
+
+    let mut worker = rust_tdlib::client::Worker::builder()
+        .with_auth_state_handler(AuthStateHandlerProxy::new_with_encryption_key("".into()))
+        .build()?;
+    worker.start();
+    let tdlib_client = worker.bind_client(tdlib_client).await?;
+
+    let client = reqwest::Client::new();
+
+    let state = AppState {
+        client,
+        tdlib_client: Arc::new(tdlib_client),
+        discord_bot_token: Arc::new(app.discord_bot_token),
+        database: Arc::new(database),
+    };
+
+    tracing::info!("Starting server...");
+    let router = Router::new()
+        .route(
+            "/verifications/telegram/:id",
+            get(handle_get_verifications::<Telegram>),
+        )
+        .route(
+            "/verifications/discord/:id",
+            get(handle_get_verifications::<Discord>),
+        )
+        .with_state(state);
+
+    let socket = SocketAddr::new(IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)), app.api_port);
```

Comment:
The entire address should be taken as a `listen-address`.

Reply:
Will be fixed, when #17 is eventually merged.